{"name": "test_login[case3]", "status": "broken", "statusDetails": {"message": "TypeError: write() argument must be str, not dict", "trace": "self = <testcase.test_login_demo.TestLogin object at 0x0000028B49D1E630>\ncase = ['不传入密码', 'password=,account=user1', '1001', '账号或密码错误', '']\n\n    def test_login(self,case):\n        url = data[1][1]  # 接口URL\n        method = data[2][1]  #请求方法\n        d = (case[1])   #获取传入接口的参数\n        r = request(url=url,method=method,data=d)\n        assert  int(case[2]) == r.json()['msg_code']  #断言响应码\n        assert case[3] == r.json()['msg']  #断言响应信息\n        test_report = {}\n        test_report['测试接口名称'] = str(case[0])\n        msg = (r.text).find(case[2])\n        if msg > 0:\n            print(case[0]+'测试成功')\n            test_report['测试结论'] = case[0]+'测试成功'\n        else:\n            print(case[0]+'测试失败')\n            test_report['测试结论'] = case[0]+'测试失败'\n        test_report['测试响应结果'] = str(r.text)\n    \n        with open(data1,'w') as f:\n>           f.write(test_report)\nE           TypeError: write() argument must be str, not dict\n\ntest_login_demo.py:38: TypeError"}, "parameters": [{"name": "case", "value": "['不传入密码', 'password=,account=user1', '1001', '账号或密码错误', '']"}], "start": 1656755939268, "stop": 1656755939271, "uuid": "1ff62e95-1664-4166-9d24-efb8650fa9d7", "historyId": "a2b6d1bc8b43b024ed347ac69e1e5c7d", "testCaseId": "635f167e9ec99310021d4ad82ace208f", "fullName": "test_login_demo.TestLogin#test_login", "labels": [{"name": "tag", "value": "@pytest.mark.skipif(False, reason='此版本不测试')"}, {"name": "suite", "value": "test_login_demo"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "22332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_demo"}]}